apstools/devices/tests/test_positioner_soft_done.py

```diff
@@ -362,3 +364,30 @@ def test_move_calcpos(target, calcpos):
     assert calcpos.inposition, str(pos)
 
     time.sleep(0.2)  # pause between tests
+
+
+@pytest.mark.parametrize(
+    # fmt: off
+    "target", [-1.2345, -1, -1, -0.1, -0.1, 0, 0, 0, 0.1, 0.1, 1, 1, 1.2345]
+    # fmt: on
+)
+def test_same_position_725(target, calcpos):
+    # Before anything is changed.
+    confirm_in_position(calcpos)
+
+    # Confirm the initial position is as expected.
+    if calcpos.position == target:
+        # First, move away from the target.
+        status = calcpos.move(-target or 0.1 * (1 + random.random()))
+        assert status.elapsed > 0, str(status)
+        confirm_in_position(calcpos)
+
+    # Move to the target position.
+    status = calcpos.move(target)
+    assert status.elapsed > 0, str(status)
```

Comment:
I'm not sure what the `assert str(status)` tests. Do we need to check for `status.done == True`?